src/Resources/Worker.php

```diff
@@ -102,17 +102,6 @@ class Worker extends Resource
      */
     public function delete()
     {
-        return $this->forge->deleteWorker($this->serverId, $this->siteId, $this->id);
-    }
-
-    /**
-     * Restart the given worker.
-     *
-     * @param  boolean $wait
-     * @return void
-     */
-    public function restart($wait = true)
-    {
-        return $this->forge->restartWorker($this->serverId, $this->siteId, $this->id, $wait);
+        return $this->forge->deleteRedirectRule($this->serverId, $this->siteId, $this->id);
```

Comment:
@fxkopp why did you change the methods here? This seems off?

Reply:
It really seems! Let me take a look tomorrow, made these changes a long time ago.

Reply:
@themsaid has reverted this: https://github.com/laravel/forge-sdk/commit/d62ecb4b654b0fa5db1dc5e8cb0131bb1ef92d27

Will release tomorrow.

Reply:
Thanks! Seems like I had the worker file open in my editor and mixed it up with the Rule file. Sorry again 😬

Reply:
@driesvints @themsaid Also fixed the incorrect delete action in the resource class, see #91